includes/class-wp-smime.php

```diff
@@ -142,8 +159,14 @@ public static function encrypt ( $message, $headers, $certificates ) {
         if ( $smime ) {
             $parts = explode( "\n\n", $smime, 2 );
             $r = array(
-                'headers' => $parts[0],
-                'message' => $parts[1],
+	            // replace for improved compatibility (e.g. Roundcube)
```

Comment:
Is this related to the fix described in the PR discussion? If not, can you open a separate PR for this merge? I'd like to investigate this separately. Thanks.

Reply:
Any danger in simply adding the RFC-conformant header instead of replacing PHP's output? If not, let's check for the existence of the new header and add it if it does not already exist. That would not molest PHP's output, and whenever it catches up your code would be a (correct) no-op.